pages/10.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

```diff
@@ -316,14 +325,19 @@ If the tracking doesn't work, take a look at the [troubleshooting section][troub
 
 [Google Analytics ID]: <https://support.google.com/analytics/answer/1008080?hl=en#trackingID>
 [Facebook Pixel ID]: <https://www.facebook.com/business/help/952192354843755>
+[Google Tag Manager]: <https://support.google.com/tagmanager/answer/6103696?hl=en#install>
 [Google Analytics USERID]: <https://support.google.com/analytics/answer/3123662?hl=en>
 [Facebook Pixel Advanced Matching]: <https://developers.facebook.com/ads/blog/post/2016/05/31/advanced-matching-pixel/> 
 [Google Analytics]: <https://developers.google.com/analytics/devguides/collection/analyticsjs/events>
 [Facebook Pixel]: <https://developers.facebook.com/docs/ads-for-websites/pixel-events/> 
 [Tag assistant]: <https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk>
 [Facebook Pixel Helper]: <https://chrome.google.com/webstore/detail/facebook-pixel-helper/fdgfkebogiimcoedlicjlajpkdmockpc>
+[Facebook Pixel Helper Img]: <media/facebook-pixel-helper.png>
```

Comment:
The images are in the same directory as the file, and we don't need to refer to them in a link macro.

```suggestion
```

Reply:
Done